website/src/app.jsx

```diff
@@ -11,6 +11,12 @@ export default React.createClass({ // eslint-disable-line
     return Object.assign({}, initialState)
   },
 
+  componentWillUpdate() {
+    const {refs: {maskedInput}} = this
+
+    maskedInput.setState(maskedInput.getInitialState())
```

Comment:
Hmm still not sure about this for the website? Why not just make things controlled and avoid this all together?